probe/probe.go

```diff
@@ -764,7 +764,12 @@ func getAnnotatedPods(c *k8s.Client, o Options, postureData map[string]string) (
 			if err != nil {
 				return nil, [][]string{}, err
 			}
-			data = append(data, []string{armoredPod.Namespace, mp[armoredPod.Namespace].NsPostureString, mp[armoredPod.Namespace].NsVisibilityString, armoredPod.Name, ""})
+			nsPostureString, nsVisibilityString := "", ""
+			if nsData, exists := mp[armoredPod.Namespace]; exists {
```

Comment:
```suggestion
			if nsData, exists := mp[p.Namespace]; exists {
```
This might be out of scope of the issue, but can we check why are we getting details of the armoredPod seperately instead of using the pod object passed down in the for loop?